pyndulum/pendulum_equations.py

```diff
@@ -2,6 +2,7 @@
 
 import numpy as np
 
+g = 9.81
```

Comment:
Note that in Python we tend to use capitals to denote constants, I suggest the following, though the rest of the file will need updating accordingly.

```suggestion
GRAV = 9.81
```